code/modules/admin/sql_message_system.dm

```diff
@@ -374,18 +374,4 @@ proc/get_message_output(type, target_ckey)
 	notesfile.cd = "/"
 	notesfile.dir.Remove(ckey)
 
-/*alternatively this proc can be run once to pass through every note and attempt to convert it before deleting the file, if done then AUTOCONVERT_NOTES should be turned off
-this proc can take several minutes to execute fully if converting and cause DD to hang if converting a lot of notes; it's not advised to do so while a server is live
-/proc/mass_convert_notes()
```

Comment:
This probably needs to stay because >legacy support.
(although idk we could drop it by now but you'd should remove the calls to it too then)